dcos/config_test.go

```diff
@@ -9,11 +9,18 @@ import (
 	"testing"
 	"time"
 
-	"github.com/pelletier/go-toml"
+	homedir "github.com/mitchellh/go-homedir"
+	toml "github.com/pelletier/go-toml"
 	"github.com/spf13/afero"
 	"github.com/stretchr/testify/require"
 )
 
+func TestMain(m *testing.M) {
+	homedir.DisableCache = true
+	// call flag.Parse() here if TestMain uses flags
+	os.Exit(m.Run())
+}
```

Comment:
Why do we need this?

Reply:
sorry stole this from my archived branch. But as there is only one or two tests changing something about `homedir` this could be done in the testitself.